resources/views/social/contents/body.blade.php

```diff
@@ -0,0 +1,96 @@
+<div class="container p-0 bg-white">
+    <a href="#">
+        <img src="https://images.pexels.com/photos/1435903/pexels-photo-1435903.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1" alt="italian" class="w-100">
```

Comment:
Is this temporary? are you gonna upload your own image?

Reply:
This is temporary image for UI design.
User will post  own image.